### PR TITLE
Grouped Combobox

### PR DESCRIPTION
**Pull request by fspoettel**

### CODE REVIEW

File: src/components/GroupedCombobox/__tests__/GroupedCombobox.spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import GroupedCombobox from '../GroupedCombobox';
+
+const getComponent = (props = {}) => shallow(
```
Comment (fspoettel) [inline src/components/GroupedCombobox/__tests__/GroupedCombobox.spec.js]: TODO: Remove

File: src/components/Combobox/components/ComboboxDropdown.js [hunk 2]
```diff
@@ -55,6 +58,32 @@ const ComboboxDropdown = React.forwardRef((props, ref) => {
...
+            id={listboxId}
+            ref={ref}
+            role="listbox"
+            tabIndex="-1"
+          >
+            {children}
+          </div>
+        </div>
+      </div>
+    </Fragment>
```
Comment (quadreex) [inline src/components/Combobox/components/ComboboxDropdown.js]: Why do we need `Fragment` here?

File: src/components/GroupedCombobox/GroupedComboboxContainer.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    const {
+      id,
+      isOpen,
+      items,
+      onToggle,
+      selectedItems
+    } = this.props;
+
+    return (
+      <Picklist
```
Comment (quadreex) [inline src/components/GroupedCombobox/GroupedComboboxContainer.js]: Haven't we deprecated `Picklist` component?

File: src/components/Combobox/components/ComboboxDropdown.js (hunk 2 above)
Comment (fspoettel) [reply src/components/Combobox/components/ComboboxDropdown.js]: Good find. We actually don't, it's a leftover from an earlier iteration. Will remove

File: src/components/GroupedCombobox/GroupedComboboxContainer.js (hunk 3 above)
Comment (fspoettel) [reply src/components/GroupedCombobox/GroupedComboboxContainer.js]: Yes and no.`Picklist` currently is a reexport of `BaseCombobox` as intended in the LDS. `Picklist` seems to be the official name of the component. `Picklist` was its own beast back before `Comboboxes` where a thing in the LDS and _that_ impl. has been removed from this project with the last major version

Comment (quadreex) [reply src/components/GroupedCombobox/GroupedComboboxContainer.js]: Shouldn't we just use it here as `BaseCombobox`? IMHO it will be more clean

Comment (fspoettel) [reply src/components/GroupedCombobox/GroupedComboboxContainer.js]: Yes, I will change it

**Pull request merged**